tests/test_transform_rule_parser.py

```diff
@@ -14,7 +14,8 @@ def test_transform_rule_parser(self, test_data_dir):
             '--ontologies-root', f'{test_data_dir}/../../ontologies/',
             '--output', output,
             '--data-source-code', 'abc',
-            '--kgiri-base', 'https://kg.your-company.kom'
+            '--kgiri-base', 'https://kg.your-company.kom',
```

Comment:
For readability of the test it would be helpful to assign `https://kg.your-company.kom` to a val and string interpolate the below SPARQL so the expected change is more obvious.

Reply:
yeah, fair point will pull these into variables and interpolate across the below stuff as well